packages/patterns-react/src/patterns/LeadSpaceCentered/__stories__/LeadSpaceCentered.stories.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { ArrowRight20, ArrowDown20, Pdf20 } from '@carbon/icons-react';
+import {
+  withKnobs,
+  text,
+  select,
+  object,
+  boolean,
+} from '@storybook/addon-knobs';
+import '../../../../../styles/scss/patterns/leadspace-centered/index.scss';
```

Comment:
@kennylam not sure where you landed, are we just including `ibm-dotcom-styles.scss` globally in storybook?
@annawen1 not sure if I saw the leadspace centered scss included on the global scss?

Reply:
Adding it now to global scss - gonna confirm with kenny if we're including `ibm-dotcom-styles.scss` globally in storybook

Reply:
talking with @kennylam, we're not going with the global scss file approach - we're instead creating an index.scss file and importing the component scss file there